src/SB/Game/zNPCFXCinematic.cpp

```diff
@@ -10,6 +10,16 @@ void zNPCFXShutdown()
 {
 }
 
+static S32 init = 0;
```

Comment:
Conventionally, this project tries to keep static variables defined in the file near the top of the file above all of the functions so that it's easy to see at a glance all the static data used by the file and verify matchingness. Please move this up to the top of the file.